src/image/ImageAnnotator.ts

```diff
@@ -99,15 +99,17 @@ export default class ImageAnnotator extends BaseAnnotator {
         const annotation = getAnnotation(this.store.getState(), annotationId);
         const annotationLocation = annotation?.target.location.value ?? 1;
         const referenceEl = this.getReference();
+        const rotation = getRotation(this.store.getState()) || 0;
 
         if (!annotation || !annotationLocation || !referenceEl || !this.annotatedEl) {
             return;
         }
 
         if (isRegion(annotation)) {
-            // TODO: Add support for scroll when image is rotated
+            const transformedShape = getTransformedShape(annotation.target.shape, rotation);
+
             scrollToLocation(this.annotatedEl, referenceEl, {
-                offsets: centerRegion(annotation.target.shape),
+                offsets: centerRegion(transformedShape),
```

Comment:
Can `centerRegion` accept an optional rotation argument and transform the shape transparently?

Reply:
It could, but it complicates what `centerRegion` does, I'd prefer to keep it explicit that we're applying a transformation to the shape before centering on it